cmd/export-distro/main.go

```diff
@@ -22,10 +23,10 @@ import (
 	"github.com/edgexfoundry/edgex-go/internal"
 	"github.com/edgexfoundry/edgex-go/internal/export/distro"
 	"github.com/edgexfoundry/edgex-go/internal/pkg/correlation"
-	"github.com/edgexfoundry/edgex-go/internal/pkg/correlation/models"
 	"github.com/edgexfoundry/edgex-go/internal/pkg/startup"
 	"github.com/edgexfoundry/edgex-go/internal/pkg/usage"
-	"github.com/edgexfoundry/go-mod-core-contracts/clients/logging"
+	logger "github.com/edgexfoundry/go-mod-core-contracts/clients/logging"
```

Comment:
Imports should be structured like this:
```
std library

external packages

internal packages
````
Please rearrange the imports.

Reply:
will do